datahub-actions/src/datahub_actions/plugin/action/metadata_change_sync/metadata_change_sync.py

```diff
@@ -21,6 +21,7 @@ class MetadataChangeEmitterConfig(BaseModel):
     gms_server: Optional[str]
     gms_auth_token: Optional[str]
     aspects_to_exclude: Optional[List]
+    entity_type_to_exclude: Optional[List] = Field(default_factory=list)
```

Comment:
if we have a default, it doesn't also need to be optional

```suggestion
    entity_type_to_exclude: List[str] = Field(default_factory=list)
```